extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -613,6 +613,29 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
     return getCsiAmpAnalyticsConfig();
   }
 
+  /** @override */
+  letCreativeTriggerRenderStart() {
+    if (
+      this.element &&
+      this.element.parentElement &&
+      this.element.parentElement.tagName == 'AMP-STICKY-AD'
+    ) {
+      const stickyMsgListener = event => {
+        if (
+          event['data'] == 'fill_sticky' &&
+          event['source'] == this.iframe.contentWindow
+        ) {
+          this.renderStarted();
+          this.element.parentElement.setAttribute('visible', '');
```

Comment:
I don't think you need to toggle visibility here. AMP sticky ad when it received render-start signal will toggle visibility. `this.renderStarted()` should have done the work.